articles/api/management/v2/tokens.md

```diff
@@ -1,86 +1,278 @@
+This will bring you to the [APIs section](${manage_url}/#/apis), specifically to the _Non Interactive Clients_ tab of the `Auth0 Management API`.
 
-Toggle the slider to authorize your client.
+![Authorize Non Interactive Client](/media/articles/api/tokens/authorize-noninteractive.png)
 
-**Choose the scopes**
+Toggle the slider from `Unauthorized` to `Authorized` for your client.
 
-You will see the available scopes that you can be granted for this client (which can be revoked at any time). Choose the desired scopes and then click the **UPDATE** button.
+### 3. Choose the Scopes
+
+The last step, before you get a token, is to select which scopes should be granted to this client.
+
+  ::: panel-info What are the scopes?
+  The scopes are permissions that should be granted by the owner. Each [Auth0 Management API v2](/api/management/v2) endpoint requires specific scopes. For example, the [Get all clients](/api/management/v2#!/Clients/get_clients) endpoint requires the scopes `read:clients` and `read:client_keys`, while the [Create a client](/api/management/v2#!/Clients/post_clients) endpoint requires the scope `create:clients`. From that we can deduce that if we need to read _and_ create clients, then our token should include three scopes: `read:clients`, `read:client_keys` and `create:clients`.
+  :::
+
+If you followed this article so far, then you already are at the _Non Interactive Clients_ tab of the `Auth0 Management API`. If not, go to [APIs](${manage_url}/#/apis), click the **Edit** icon for `Auth0 Management API`, go to _Non Interactive Clients_ and expand your non interactive client, using the pointing down arrow, next to the _Authorized_ toggle.
+
+All the scopes, for the various [Management API v2](/api/management/v2) endpoints are listed here. Tick the ones you want to grant to the client and click the **Update** button.
+
+  ::: panel-info I don't know which scopes to choose!
+  To find out which scopes you should choose, go the [Management API v2](/api/management/v2) and note the scopes listed at each endpoint. For example, the [Get all Clients](/api/management/v2#!/Clients/get_clients) endpoint requires the scopes `read:clients` and `read:client_keys`, while the [Create a Client](/api/management/v2#!/Clients/post_clients) endpoint requires the scope `create:clients`. If your client needs to read and create users, then you need to tick the checkboxes for these three scopes: `read:clients`, `read:client_keys` and `create:clients`.
+  :::
 
 ![Choose authorized scopes](/media/articles/api/tokens/choose-scopes.png)
 
-**Get the token**
+### 4. Get the Token
 
-To test the client interaction with the API, go to the **Test** section under **Auth0 Management API**.
+To get a token, go to the *Test* section under `Auth0 Management API`.
+
+This page will give you code snippets on how to form a request to get a token.
+
+Check the _Response_, a token is already generated for you. Click **Copy Token**.
```

Comment:
Hmm interesting, I thought it was there for all tenants. I will check with crew-2.

This is what I see:

![image](https://cloud.githubusercontent.com/assets/11715799/22932572/120e8738-f2d1-11e6-9c79-0c563cd6cb7f.png)